.github/workflows/update-Dockerfiles.yml

```diff
@@ -127,7 +129,7 @@ jobs:
         uses: repo-sync/pull-request@v2
         with:
           source_branch: ${{ steps.commit-push.outputs.BRANCH }}
-          destination_branch: "master"
+          destination_branch: "dev"
```

Comment:
Checking my understanding, this PR will now target `dev`, so when releasing the images we'll have to do the `dev` -> `master` sync that we don't today? But we'll minimize those master-only commits that skip dev?

Reply:
Correct. I am trying to force everything to go through `dev` first. This will align with our future efforts to automate dev->main syncs.